web/components/ResourceAddButton.vue

```diff
@@ -114,6 +124,11 @@ export default defineComponent({
           // if store.count = 0, name suffix is 1.
           targetTemplate = storageclassTemplate((store.count + 1).toString());
           break;
+        case "PriorityClass":
+          store = priorityclassstore;
+          // if store.count = 0, name suffix is 1.
+          targetTemplate = priorityclassTemplate((store.count + 1).toString());
```

Comment:
There are two Priority classes from the beginning. ("system-cluster-critical" and "system-node-critical")
> Kubernetes already ships with two PriorityClasses: system-cluster-critical and system-node-critical. These are common classes and are used to ensure that critical components are always scheduled first.
https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/

```suggestion
          // if store.count = 2, name suffix is 1.
          targetTemplate = priorityclassTemplate((store.count - 1).toString());
```